docker/bin/stop-presto

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# This works for worker only. Coordinator doesn't support graceful shutdown.
+# This script will block until the server has actually shutdown.
+set -x
+
+http_port="$(cat /usr/lib/presto/etc/config.properties | grep 'http-server.http.port' | sed 's/^.*=\(.*\)$/\1/')"
+https_port="$(cat /usr/lib/presto/etc/config.properties | grep 'http-server.https.port' | sed 's/^.*=\(.*\)$/\1/')"
+
+if [ -n "$http_port" ] ; then
+    res=$(curl -s -o /dev/null -w "%{http_code}"  -XPUT --data '"SHUTTING_DOWN"' -H "Content-type: application/json" http://localhost:${http_port}/v1/info/state)
```

Comment:
- Please use long option names.
- also add `--show-error --fail`
- quote variables in `"`

Reply:
For `curl`, you want to use `-sS` rather than just `-s` so that it prints an error on failure.

Reply:
Fixed. Now using `--silent --show-error` (i.e. `-sS`)